Tests/AssemblerLib/LocalToGlobalIndexMap.cpp

```diff
@@ -51,34 +54,35 @@ class AssemblerLibLocalToGlobalIndexMapTest : public ::testing::Test
     //data component 0 and 1 are assigned to all nodes in the mesh
     static std::size_t const comp0_id = 0;
     static std::size_t const comp1_id = 1;
-    std::vector<MeshLib::MeshSubsets*> components;
+    std::vector<std::unique_ptr<MeshLib::MeshSubsets>> components;
+    std::size_t components_size;
```

Comment:
Moved components_size to the test body, because it is local their. The name is then correct.